python/dist2wheel.py

```diff
@@ -73,33 +75,51 @@ def create_all(self):
         for artifact in self.artifacts:
             try:
                 self.path = artifact["path"]
-                self.platform_tag = self._fixup_platform_tag(artifact)
+                self._set_platform_props(artifact)
                 self.checksum = self._fix_checksum(artifact["extra"]["Checksum"])
                 self.size = os.path.getsize(self.path)
             except KeyError:
                 continue
 
             self.wheel_file = WHEEL_TEMPLATE.format(**self.__dict__).encode()
             self.metadata_file = METADATA_TEMPLATE.format(**self.__dict__).encode()
+            os.makedirs(self.wheel_dir, exist_ok=True)
             self._emit()
 
-    @staticmethod
-    def _fixup_platform_tag(artifact):
+    def _matrix(self):
+        return {
+            "darwin": {
+                "amd64": {
+                    "arch": "x86_64",
+                    "platform": "macosx_10_7_x86_64",
```

Comment:
Go 1.21 claims that `macosx_10_15` is the minimum supported version.  And it's unwise from a security posture to compile with anything less than the current version.

Reply:
Sure, I can totally bump it.  I figured since we aren't compiling it directly on MacOS as part of the release process, it wasn't as much of a concern, merely packaging up pre-built binaries into a wheel.

Reply:
updated